.travis.yml

```diff
@@ -78,15 +78,21 @@ jobs:
         - node build-system/pr-check/remote-tests.js
       after_script:
         - build-system/sauce_connect/stop_sauce_connect.sh
+    - stage: test
+      name: "End to End Tests"
+      script:
+        - node build-system/pr-check/e2e-tests.js
   fast_finish: true
 
   # TODO(ampproject/wg-infra): remove when remote tests stabilize
   allow_failures:
     - script:
       - node build-system/pr-check/remote-tests.js
+      - node build-system/pr-check/e2e-tests.js
```

Comment:
FYI things are looking better - I moved the tests to its own shard and fixed the chromedriver process that was causing the Travis VM to hang. But I haven't gotten a consistent green test run yet, and it looks like the test runner isn't running all tests or reporting them correctly, based on the latest result - `Executed 88 of 100 (Skipped 15) 23 FAILED `

We should aim for all tests to pass before turning this on, right?

Reply:
Some of the `amp-list` tests are flaky and @cathyxz is investigating. The `amp-carousel` tests should be more consistently green

Reply:
Got it! I'm going to merge this in nonblocking mode, and let's reduce the flakes in future PRs.